sfge/src/main/java/com/salesforce/config/EnvUtil.java

```diff
@@ -47,12 +47,12 @@ static long getRuleThreadTimeout() {
     /**
      * Indicates if info-level violations are enabled
      *
-     * @return value of {@link #ENV_RULE_ENABLE_WARNING_VIOLATION} env variable. If it is not set,
-     *     {@link #DEFAULT_RULE_ENABLE_WARNING_VIOLATION}
+     * @return value of {@link #ENV_RULE_DISABLE_WARNING_VIOLATION} env variable. If it is not set,
+     *     {@link #DEFAULT_RULE_DISABLE_WARNING_VIOLATION}
```

Comment:
Do we need a comment negation?

Reply:
No, because it still returns true when the env-var is true.